scripts/init_llm_models.py

```
@@ -0,0 +1,1152 @@
+        llm_context_window=200_000,
+        llm_max_output_tokens=100_000,
+        llm_is_vision_model=True,
+        llm_is_thinking_model=True,
+        llm_supports_json=True,
+        llm_supports_temperature=False,
+        is_deprecated=True,
+        redirect_to=o3,
+    )
+
+    # https://platform.openai.com/docs/models#o1
+    o1_preview = AIModelSpec.objects.create(
```

Ruff (0.14.10)

Local variable `o1_preview` is assigned to but never used

Remove assignment to unused variable `o1_preview`

(F841)